packages/cozy-harvest-lib/src/components/AccountSelectBox/AccountSelectBox.jsx

```diff
@@ -12,8 +12,12 @@ const AccountSelectBox = ({
   accountsList,
   selectedAccount,
   onChange,
-  onCreate
+  onCreate,
+  loading
 }) => {
+  if (loading) {
+    return <div style={{ height: 20.8 }}>&nbsp;</div>
```

Comment:
tiny is about 2rem, no? https://github.com/cozy/cozy-ui/blob/master/react/SelectBox/SelectBox.jsx#L14 Weird to see 20.8px

Reply:
I've checked the height of the component and the height was 20.8 exactly. It's true that it is weird :/